docs/variables/gamesplayed.md

```diff
@@ -0,0 +1,35 @@
+---
+id: gamesplayed
+---
+
+# $(gamesplayed)
+
+Returns a list of recorded directories that a broadcaster's latest stream was, or current stream is under.
+
+:::info This feature is managed by a custom tracking system!
+
+Fossabot only populates this list if it was part of your channel during at least one stream.
```

Comment:
*one game change

I think it actually stores the first game by default (but sets started_at to be roughly when you registered since it's an async worker, I could be wrong but i think fossa pulls that as part of user hydration on signup)

Reply:
I'll change it to `game change` and leave this description for now, so in case it's not pulled on sign up people can find an explanation for it.